src/Avalonia.Base/Media/TextFormatting/Unicode/EastAsianWidthClass.cs

```diff
@@ -0,0 +1,12 @@
+namespace Avalonia.Media.TextFormatting.Unicode
+{
+    public enum EastAsianWidthClass
```

Comment:
Do we want to keep this namespace public?

Reply:
It is exposed by Codepoint